docs/pages/build/automate-submissions.mdx

```diff
@@ -47,3 +51,39 @@ export default () => {
 ```
 
 The `APP_ENV` variable from the `production` profile will be used when evaluating **app.config.js** for the submission, and therefore the name will be `My App` and the bundle identifier will be `com.my.app`.
+
+## Default submission behavior for app stores
+
+Understanding the default behavior of `--auto-submit` is important for planning your release process. Sections below describe the default submission behavior for Android and iOS.
```

Comment:
Assuming my comments below about the default behavior are accurate, I'd recommend a TL;DR headline here like: "By default, the `--auto-submit` lag will make your build available for internal testing, but will not automatically submit to your app to review for public distribution."